pyimgbatch/__main__.py

```diff
@@ -7,15 +8,17 @@
 def main():
     """simply main
     """
-    logging.basicConfig(level=logging.DEBUG,
-                        filename='pyimgbatch.log',
+    args = get_args()
+    logging_level = logging.DEBUG if not args.nolog else logging.WARNING
```

Comment:
Why you set it to warning if the arg is set to nolog?

Reply:
Should i config it to `ERROR` or `CRITICAL`  ?

Reply:
ERROR 
and thanks for your work